libs/ebpfnetsh/pins.cpp

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#define WIN32_LEAN_AND_MEAN
+#include <windows.h>
+
+#include <iostream>
+#include <iomanip>
+#include <netsh.h>
+#include <set>
+#include <string>
+#include <vector>
+#include "bpf.h"
+#include "ebpf_api.h"
+#include "ebpf_windows.h"
+#include "pins.h"
+#include "platform.h"
+#include "tokens.h"
+
+DWORD
+handle_ebpf_show_pins(
+    LPCWSTR machine, LPWSTR* argv, DWORD current_index, DWORD argc, DWORD flags, LPCVOID data, BOOL* done)
+{
+    UNREFERENCED_PARAMETER(machine);
+    UNREFERENCED_PARAMETER(flags);
+    UNREFERENCED_PARAMETER(data);
+    UNREFERENCED_PARAMETER(done);
+
+    std::cout << "\n";
+    std::cout << "     ID     Type  Path\n";
+    std::cout << "=======  =======  ==============\n";
+
+    // Read all pin paths.  Currently we get them in a non-deterministic
+    // order, so we use a std::set to sort them lexicographically.
```

Comment:
Technically this isn't strictly lexicographical order either when dealing with UTF-8 characters sequences. Multi-point code sequences won't get sorted into lexigraphic order. I don't think it matters, but the comment isn't strictly true.

Reply:
See https://en.cppreference.com/w/cpp/locale/locale/operator() for better option for sorting in lexographical order.

Reply:
Since the netsh helper isn't internationalized yet, I am changing the comment to "code point order" which is what it's actually called as is.

Reply:
Issue #558